shared/xccdf/services/base.xml

```diff
@@ -117,7 +117,7 @@ multiple processors.
 <ocil><systemd-check-macro enable="false" service="irqbalance" /></ocil>
 <rationale>In an environment with multiple processors (now common), the irqbalance service
 provides potential speedups for handling interrupt requests.</rationale>
-<platform idref="cpe:/a:machine"/>
+<machine />
```

Comment:
I am not a big fan of this syntax. It's not self-documenting at all. It doesn't allow to set some rules as container only. It doesn't allow anything for VMs.